README.md

```diff
@@ -1,38 +1,40 @@
+In this project, you will explore handling input events and submission by creating a [Mad Lib](https://en.wikipedia.org/wiki/Mad_Libs) game.
 
-This is a [Stage]() project. You will be submitting a PR for this project at the end.
+This is a [Stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-2) individual project. You will be submitting a PR for this project at the end.
 
 ## Learning Goals
-- Extend code provided
 - Practice using events
 - Pass custom information to an event handler function
 - Dynamically construct components based on data
+- Extend code provided
 
 ## Scaffolding
-The data structure that drives the UI is madlibs/MadLibs.js. To complete your objectives, you will not need to modify this file. The data in this file is an array of Madlib objects. A MadLib object has three properties that you'll utilize:
+The data structure that drives the UI is madlibs/MadLibs.js. To complete your requirements, you will not need to modify this file. The data in this file is an array of Madlib objects. A MadLib object has three properties that you'll utilize:
 - `title` - the title of the Madlib
-- `words` - an array of fill-in-the-blank 'word' objects (these objects contain a `key`, `label`, and `value` property)
-- `getText()` - a function that returns the Madlib text
+- `words` - an array of fill-in-the-blank 'word' objects. Each object contain a `key`, `label`, and `value` property. You will fill in the values
+- `getText()` - a function that returns the Madlib text with each word filled in
 
 ## Requirements
 
 ### Baseline
 As a starting point, review the code that has been provided to you.
 
-To work through the analysis of the existing code, determine how to get the application to load a **random** MadLib, instead of loading the first one which it will currently always do by default.
+To work through the analysis of the existing code, determine how to modify the application to load a **random** MadLib, instead of loading the first one which it currently does by default.
 
 ### Wave 1
-* `WordForm.js` should render text inputs for all word objects in the array - use the `label` property in the word object for the value of the input's [placeholder](http://www.w3schools.com/Tags/att_input_placeholder.asp) attribute.
-  * These text inputs should call the `_onInputChange()` function in `WordForm.js` when a change event occurs - and pass in the appropriate arguments that `_onInputChange()` is expecting.
-* The `Story.js` component should simply display the value returned from the `getText()` function on the `MadLib` object - but only have this component displayed when all word objects have values.
+1. Create a new component which will render the forms inputs for each word in the puzzle.
+  - You should utilize the `selectedMadLib` variable stored in the `App` components `state` to gather the words you need.
```

Comment:
These sub-bullet points will need to be indented with four spaces in order to keep them from resetting the ordered list items.